cloud_Azure/terraform/module/storage_account.tf

```diff
@@ -12,4 +12,4 @@ resource "azurerm_storage_account" "logs_storage_account" {
   tags = {
     app = "kentik_flow_log_exporter"
```

Comment:
The tag should be configurable ("flow_flow_log_exporter" is a good default).

Reply:
adding variable "resource_tag" with default of "flow_log_exporter"